ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/ButtonComponentStyle.kt

```diff
@@ -0,0 +1,33 @@
+package com.revenuecat.purchases.ui.revenuecatui.components.style
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import com.revenuecat.purchases.ui.revenuecatui.components.PaywallAction
+
+@Immutable
+internal class ButtonComponentStyle private constructor(
+    @get:JvmSynthetic
+    val stackComponentStyle: StackComponentStyle,
+    @get:JvmSynthetic
+    val action: PaywallAction,
+    @get:JvmSynthetic
+    val actionHandler: (PaywallAction) -> Unit,
```

Comment:
I've been wondering as well whether it would make sense to remove the `action` parameter and have a single lambda passed in that would be called when the button is clicked... But I think this provides more flexibility in case we need to do anything in the button itself.

Reply:
I think this is fine! Don't really have an opinion yet I think haha. We can go with this and always change it if we see that the other option would've been better.